hnn_core/cells_default.py

```diff
@@ -429,6 +429,7 @@ def pyramidal_ca(cell_name, pos, override_params=None, gid=None):
     override_params['L5Pyr_dend_gbar_ca'] = gbar_ca
     override_params['L5Pyr_dend_gnabar_hh2'] = gbar_na
     override_params['L5Pyr_dend_gkbar_hh2'] = gbar_k
+    override_params['L5Pyr_soma_gbar_ca'] = 10.
```

Comment:
separate PR for separate problems please!

Reply:
I think I pushed this a while ago??

Reply:
Oh yeah, this is #891

The calcium model is not implemented correctly right now ... (at least not the way Sarah did it).

Reply:
that's fine! Just remove the change from this PR